game_eggs/gta/openmp/config.json

```diff
@@ -0,0 +1,93 @@
+{

```

Comment:
is there any reason this is on .json wile there docs mention .cfg?
https://www.open.mp/docs/server/server.cfg

Reply:
Docs are for SA-MP, maybe somewhere on open.mp there is documentation for config.json, but open.mp server won't work without config.json, server.cfg isn't supported